recipe-client-addon/test/unit/mock_api/api/v1/action/index.json

```diff
@@ -15,12 +15,44 @@
       "title": "Log a message to the console",
       "type": "object"
     },
-    "implementation_url": "https://localhost:8000/api/v1/action/console-log/implementation/0765c6302db2846b85543eea743f959c19b42559/",
+    "implementation_url": "https://localhost:8443/api/v1/action/console-log/implementation/sha384-RGx3rydrSq53UfmW9kFcK0mQYra67XIvZvr4MhmAe--ljiiMQOtgM7Cmca48um3v/",
     "name": "console-log"
   },
   {
     "arguments_schema": {
       "$schema": "http://json-schema.org/draft-04/schema#",
+      "properties": {
+        "addonUrl": {
+          "description": "URL of the add-on XPI file",
+          "format": "uri",
+          "minLength": 1,
+          "type": "string"
+        },
+        "description": {
+          "description": "User-facing description of the study",
+          "minLength": 1,
+          "type": "string"
+        },
+        "name": {
+          "description": "User-facing name of the study",
+          "minLength": 1,
+          "type": "string"
+        }
+      },
+      "required": [
+        "name",
+        "description",
+        "addonUrl"
+      ],
+      "title": "Enroll a user in an opt-out SHIELD study",
+      "type": "object"
+    },
+    "implementation_url": "https://localhost:8443/api/v1/action/opt-out-study/implementation/sha384-HM_avYcD00o27ufwU1V7PIBtiuMAXML6MMwlYrDEqDX-XzGVuOfL52RCM680JExN/",
```

Comment:
As mentioned above, I think it would be cleaner to split this into
```json
{
"implementation_url": "https://localhost:8443/api/v1/action/opt-out-study/implementation/foo",
"implementation_hash": "HM_avYcD00o27ufwU1V7PIBtiuMAXML6MMwlYrDEqDX-XzGVuOfL52RCM680JExN",
}
```
which, in passing, doesn't really need to be base64url, since it's not in a url.